auth-connector/src/main/java/com/sequenceiq/cloudbreak/auth/altus/model/Entitlement.java

```diff
@@ -119,5 +119,7 @@ public enum Entitlement {
     CDP_POSTGRES_UPGRADE_SKIP_ATTACHED_DATAHUBS_CHECK,
     ACCESS_KEY_ECDSA,
     CDP_ENVIRONMENT_EDIT_PROXY_CONFIG,
-    SDX_CONFIGURATION_OPTIMIZATION
+    SDX_CONFIGURATION_OPTIMIZATION,
+
```

Comment:
why we need the extra line here?

Reply:
Ah, it was a conflict during rebase, what I have resolved incorrectly. Fixing it now.